pom.xml

```diff
@@ -178,6 +178,8 @@
             <!-- # Allow loading dynamic agent used by JOL -->
             -XX:+EnableDynamicAgentLoading
             ${extraJavaVectorArgs}
+            <!-- # https://bugs.openjdk.org/browse/JDK-8327134 -->
+            -Djava.security.manager=allow
```

Comment:
we should add it only in modules where it's actually needed IMO.
Otherwise it's easy to introduce new usages of stuff which requires this flag.

I'm sending a follow-up PR.

Reply:
In my discussion with @martint we kinda concluded that whatever is required for Trino in general should be injected by the launcher as setting .. for example also 

-Djdk.attach.allowAttachSelf=true

And then the rest of the config we have in the docs and built into the rpm and docker container are just best practices .. but not required completely.